### PR TITLE
Add Gvalchca SpaccBoard

### PR DESCRIPTION
**Pull request by Gvalch-ca**
## Description

Add Gvalch'ca SpaccBoard PCB to QMK. It is a QMK and VIA-compatible 60% PCB designed to fit NMB Hi-Tek 725 "Space Invaders" switches. PCB is distributed through GBs hosted by main designer (me, Gvalch'ca) and as in-stock item on keebmeup.com. First round of GB (20 units) was successfully finished some time ago.
Also this PR contains a new vendor folder (gvalchca) for all my works, since I have 2 more PCBs on the stage of prototyping right now.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new vendor folder - gvalchca
* Added new keyboard - gvalchca/spaccboard (QMK and VIA)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gvalchca/spaccboard/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  KC_CAPS, KC_A,    KC_S,    KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           \
+  KC_LSFT,          KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                   \
+  KC_LCTL,       KC_LGUI, KC_LALT,                         KC_SPC,                             KC_RALT, KC_RGUI, KC_RCTRL),
+
+[_FUNC] = LAYOUT(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,          \
+  _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   \
+  _______,          _______, _______,                            _______,                            _______, _______, _______)
```
Comment (fauxpark) [inline keyboards/gvalchca/spaccboard/keymaps/default/keymap.c:22]:
```suggestion
  KC_ESC,  KC_1,    KC_2,    KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_BSPC,
  KC_TAB,  KC_Q,    KC_W,    KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
  KC_CAPS, KC_A,    KC_S,    KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
  KC_LSFT,          KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
  KC_LCTL,       KC_LGUI, KC_LALT,                         KC_SPC,                             KC_RALT, KC_RGUI, KC_RCTRL),

[_FUNC] = LAYOUT(
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
  _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______,          _______, _______,                            _______,                            _______, _______, _______)
```

File: keyboards/gvalchca/spaccboard/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Gvalchca x Jugizafupe SpaccBoard
+======
+
+Space Invaders 60% PCB with ANSI and Big Ass Enter support. To update the firmware connect the PCB to your computer and press Reset button.
+![SpaccBoard image](https://i.imgur.com/ZYSrQTr.jpeg)
+
+Keyboard Maintainer: [Gvalchca](https://github.com/Gvalch-ca)  
+Hardware Supported: Gvalchca x Jugizafupe SpaccBoard
+Hardware Availability: [KeebMeUp](https://keebmeup.com)
```
Comment (fauxpark) [inline keyboards/gvalchca/spaccboard/readme.md:9]:
```suggestion
# Gvalchca x Jugizafupe SpaccBoard

Space Invaders 60% PCB with ANSI and Big Ass Enter support. To update the firmware connect the PCB to your computer and press Reset button.
![SpaccBoard image](https://i.imgur.com/ZYSrQTr.jpeg)

* Keyboard Maintainer: [Gvalchca](https://github.com/Gvalch-ca)
* Hardware Supported: Gvalchca x Jugizafupe SpaccBoard
* Hardware Availability: [KeebMeUp](https://keebmeup.com)
```

File: keyboards/gvalchca/spaccboard/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+COMMAND_ENABLE = no     # Commands for debug and configuration
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+RGBLIGHT_ENABLE = no    # Enable keyboard underlight functionality
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+UNICODE_ENABLE = yes    # Unicode
+BLUETOOTH_ENABLE = no   # Enable Bluetooth
+LTO_ENABLE = yes
+
+LAYOUTS = ansi bae
```
Comment (fauxpark) [inline keyboards/gvalchca/spaccboard/rules.mk:22]:
```suggestion
# Bootloader selection
BOOTLOADER = qmk-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
UNICODE_ENABLE = yes    # Unicode
LTO_ENABLE = yes
```

File: keyboards/gvalchca/spaccboard/spaccboard.h [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e },  \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, ___, ___ },  \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, ___, k2d, ___ },  \
+    { k30, ___, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, ___, ___ },  \
+    { k40, ___, k42, k43, ___, ___, ___, k47, ___, ___, ___, k4b, k4c, k4d, ___ }   \
+}
+
+
+void matrix_init_user(void);
+void matrix_scan_user(void);
```
Comment (fauxpark) [inline keyboards/gvalchca/spaccboard/spaccboard.h:54]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/gvalchca/spaccboard/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/gvalchca/spaccboard/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/gvalchca/spaccboard/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/gvalchca/spaccboard/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/gvalchca/spaccboard/spaccboard.c [hunk 7]
```diff
@@ -0,0 +1 @@
+#include "spaccboard.h"
```
Comment (drashna) [inline keyboards/gvalchca/spaccboard/spaccboard.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/gvalchca/spaccboard/spaccboard.h [hunk 8]
```diff
@@ -0,0 +1,54 @@
+#pragma once
```
Comment (drashna) [inline keyboards/gvalchca/spaccboard/spaccboard.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/gvalchca/spaccboard/keymaps/default/keymap.c (hunk 5 above)
Comment (Gvalch-ca) [reply keyboards/gvalchca/spaccboard/keymaps/default/keymap.c:17]: Done

File: keyboards/gvalchca/spaccboard/keymaps/via/keymap.c (hunk 6 above)
Comment (Gvalch-ca) [reply keyboards/gvalchca/spaccboard/keymaps/via/keymap.c:17]: Done

File: keyboards/gvalchca/spaccboard/spaccboard.c (hunk 7 above)
Comment (Gvalch-ca) [reply keyboards/gvalchca/spaccboard/spaccboard.c]: Done

File: keyboards/gvalchca/spaccboard/spaccboard.h (hunk 8 above)
Comment (Gvalch-ca) [reply keyboards/gvalchca/spaccboard/spaccboard.h:17]: Done

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by Erovia — APPROVED** [review]

**Pull request merged**